spec/acceptance/mod_passenger_spec.rb

```diff
@@ -1,56 +1,16 @@
 require 'spec_helper_acceptance'
 require_relative './version.rb'
 
-describe 'apache::mod::passenger class' do
-  case fact('osfamily')
-  when 'Debian'
-    conf_file = "#{$mod_dir}/passenger.conf"
-    load_file = "#{$mod_dir}/zpassenger.load"
+describe 'apache::mod::passenger class', if: fact('osfamily') == 'Debian' do
```

Comment:
I'd be nice if there were acceptance tests for RH since it's something we do rely on but right now the tests aren't executed for RH anyway, right?

Reply:
Yeh, I looked through the history of the file and found that they were removed by @tphoney

Reply:
installing the various mods in redhat is painful, hence the removal